src/liballoc/collections/btree/navigate.rs

```diff
@@ -161,15 +155,16 @@ impl<'a, K, V> Handle<NodeRef<marker::Mut<'a>, K, V, marker::Leaf>, marker::Edge
 impl<K, V> Handle<NodeRef<marker::Owned, K, V, marker::Leaf>, marker::Edge> {
     /// Moves the leaf edge handle to the next leaf edge and returns the key and value
     /// in between, while deallocating any node left behind.
-    /// Unsafe for three reasons:
+    /// Unsafe for two reasons:
     /// - The caller must ensure that the leaf edge is not the last one in the tree
     ///   and is not a handle previously resulting from counterpart `next_back_unchecked`.
-    /// - If the leaf edge is the last edge of a node, that node and possibly ancestors
+    /// - Further use of the updated leaf edge handle is very dangerous. In particular,
+    ///   if the leaf edge is the last edge of a node, that node and possibly ancestors
     ///   will be deallocated, while the reference to those nodes in the surviving ancestor
-    ///   is left dangling; thus further use of the leaf edge handle is dangerous.
-    ///   It is, however, safe to call this method again on the updated handle.
-    ///   if the two preconditions above hold.
-    /// - Using the updated handle may well invalidate the returned references.
```

Comment:
This was unrelated copy-and-paste baloney. No references were returned.

Reply:
The return type is `(&'a mut K, &'a mut V)` -- looks like returned references to me?

Reply:
It's not, it's `(K, V)`?? I would think you were right because I edited this a lot about this at the time, but then again, looking at the period behind "handle", even AI would have at least gotten the grammar right.

Reply:
Ah right, it is `(K, V)` here. But it is a pair of references in some other place where you removed that comment.